src/sentry/integrations/github/webhook.py

```diff
@@ -88,9 +87,9 @@ def _handle(self, event, organization, repo):
 class PushEventWebhook(Webhook):
     # https://developer.github.com/v3/activity/events/types/#pushevent
 
-    def _handle(self, event, organization, repo):
+    def _handle(self, integration, event, organization, repo):
         authors = {}
-        client = GitHubAppsClient(event['installation']['id'])
```

Comment:
this should be pulled out already from `_handle` but am unsure if `integration.get_installation().get_client()` should just work for GHE, it seems like it might?